metric-anomaly-detector-api/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+plugins {
+    `java-library`
+    // id("org.hypertrace.avro-plugin")
```

Comment:
```suggestion
```

Reply:
The above plugin is required - id("org.hypertrace.avro-plugin"), right?

Reply:
So this plugin performs a backward compatibility check with the last published version, so will enable it after first version artifact is published